Source/WebCore/platform/graphics/cocoa/VP9UtilitiesCocoa.mm

```diff
@@ -807,11 +807,13 @@ static uint8_t convertSubsamplingXYToChromaSubsampling(uint64_t x, uint64_t y)
     return createVideoInfoFromVPCodecConfigurationRecord(record, header.width, header.height);
 }
 
-bool hasVP9ExtensionSupport()
+bool av1HardwareDecoderAvailable()
```

Comment:
this would be better in an AV1UtilitiesCocoa.

There's VP9 and HEVC already

Reply:
Done